plugins/modules/repository_sync.py

```diff
@@ -66,9 +66,7 @@ def main():
     module = KatelloAnsibleModule(
         foreman_spec=dict(
             product=dict(type='entity', scope=['organization'], required=True),
-            repository=dict(type='entity', scope=['product'], failsafe=True),
-            # This should be scoped more explicit for better serch performance, but needs rerecording
-            # repository=dict(type='entity', scope=['organization', 'product'], failsafe=True),
+            repository=dict(type='entity', scope=['product']),
```

Comment:
I get, the `failsafe` was the issue. What about the "better scoping" was that real?

Reply:
No, the commit that dropped the comment says:
> the product is already scoped by org, so there is no benefit of scoping
> the repository by both product *and* org.